Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -90,6 +101,15 @@
       normal: ian
       crit: ian_dead
       dead: ian_dead
+  - type: InteractionPopup
+    interactSuccessString: petting-success-dog
+    interactFailureString: petting-failure-generic
+    interactSuccessSound:
+      path: /Audio/Animals/small_dog_bark_happy.ogg
```

Comment:
this shouldnt be necessary since its inherited from `MobCorgi`

Reply:
Good catch, fixing now.

Reply:
![image](https://user-images.githubusercontent.com/97707302/153341335-1889e4d4-d25e-4e53-bd13-cac4d3c148e4.png)